### ISSUE TITLE
Create Auth (Session) example

### ISSUE THREAD

**Issue by talentlessguy** [talentlessguy/tinyhttp#103]
Create an example using tinyhttp and [`@tinyhttp/session`](https://tinyhttp.v1rtl.site/mw/session) under `examples/session` folder

The app logic can be taken from JWT example

**Comment by aneeshrelan** [comment]
Can I take up this one?

**Comment by talentlessguy** [comment]
@aneeshrelan yep

**Comment by aneeshrelan** [comment]
I might need some help here @talentlessguy . 

Have create a new dummy copying the `jwt` example. With just the following code to start with
package.json
```
{
  "name": "session",
  "private": true,
  "dependencies": {
    "@tinyhttp/app": "workspace:*",
    "@tinyhttp/session": "workspace:*"
  },
  "type": "module",
  "module": "index.js"
}
```
index.js
```
import { App } from '@tinyhttp/app'

const app = new App()

app.get('/', (_req, res) => {
  res.send('Go to "/login" page to login')
})

app.listen(3000)
```

I've executed the `pnpm i && pnpm i -r` in the root of the repo and also in the `examples/session` folder. But the `node_modules` doesn't seem to come up and then running `node index.js` results in a _Module not found_ error. I think I'm missing something here

Node version: v14.13.0

**Comment by talentlessguy** [comment]
@aneeshrelan try doing `pnpm i @tinyhttp/app` in the `session` folder

**Comment by aneeshrelan** [comment]
Yup, this works. Do you think, this should be added to either the README or the CONTRIBUTING for the examples directory?

**Comment by talentlessguy** [comment]
@aneeshrelan thx for suggestion, I will add this to CONTRIBUTING.md file as a note soon

**Comment by talentlessguy** [comment]
Closed via #103
